tools/common.py

```diff
@@ -0,0 +1,10 @@
+def include_dir_order():
+  return (
+      "envoy/",
```

Comment:
Can this just be a constant set? That way we have fast hash lookup.

Reply:
Well the order is significant in one usage.  Making a dict makes sense though, not here but in the usage in check_format.py.